packages/headless/dist/parsed_doc.json

```diff
@@ -0,0 +1,209 @@
+{
+  "controllers": [ "Hidden for this PR" ],
```

Comment:
Like the previous PR, this file is only included to review the output, hence why I hid the controllers.